clipper_admin/clipper_manager.py

```diff
@@ -498,13 +501,116 @@ def deploy_model(self,
                 for r in range(num_containers)
             ])
 
+
+
+
+    def deploy_R_model(self,
+                     name,
+                     version,
+                     model_data,
+                     container_name,
+                     labels,
+                     input_type,
```

Comment:
Switch the order of `labels` and `input_type` arguments and make `labels` an optional arg with a default: `labels=DEFAULT_LABEL`